lib/gui/app/styled-components.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+const styled = require('styled-components').default
+const { colors, consts } = require('./theme')
+const {
+  Button, Txt, Flex
+} = require('rendition')
+
+exports.StepButton = styled(Button) `
+  width: 100%;
+  max-width: ${consts.btnMaxWidth};
+  margin: auto;
+  overflow: hidden;
+
+  &:disabled {
+    background-color: ${colors.dark.disabled.background};
+    color: ${colors.dark.disabled.foreground};
+    &:hover {
+      background-color: ${colors.dark.disabled.background};
+      color: ${colors.dark.disabled.foreground};
+    }
+  }
+`
+exports.ChangeButton = styled(Button) `
+  font-size: 12px;
+  color: ${colors.primary.background};
+  padding: 0;
+  width: 100%;
+
+  &:hover, &:focus, &:active {
+    color: ${colors.primary.background};
+  }
+`
+exports.StepNameButton = styled(Button) `
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 39px;
+  width: 100%;
+  font-weight: bold;
+  color: #fff;
+
+  &:hover, &:focus, &:active{
+    color: ${colors.primary.foreground};
+  }
+`
+exports.StepSelection = styled(Flex) `
+  flex-wrap: wrap;
+  justify-content: center;
+`
+exports.Footer = styled(Txt) `
+  margin-top: 10px;
+  color: #787c7f;
```

Comment:
How about referencing this from the theme as well?

Reply:
done and check for other occurences (oversight I'm afraid)